qrexec/tools/qrexec_policy_agent.py

```diff
@@ -184,11 +184,22 @@ def apply_model(
                 completion.set_inline_selection(True)
                 completion.set_inline_completion(True)
                 completion.set_popup_completion(True)
-                completion.set_popup_single_match(False)
+                completion.set_popup_single_match(True)
```

Comment:
Can you explain this change? It means you can't accept a single match (after typing it in) with a single click anymore, you need two clicks. OpenQA found this out, but TBH I'm not sure if that's a common case.

Reply:
> Can you explain this change?

That is something I found only after using the patch for a while. The default logic (matching from the start) does not require to show the popup for a single match if user has entered the entire string.

In our case, if user wants to search for `disp1234` and user has currenty `disp1234`, `dips5678` and `disp90ab` running at the moment and user types `1`, there will not popup result to select `disp1234` from it if `popup_single_match` is et to False.

Reply:
p.s. Actually the GTK default matching algorithm is also insufficient if a popup is not shown. As it is case in-senstivie and Qube names are case-sensitive.

Reply:
I see, ok.